pdal/PointView.hpp

```diff
@@ -182,19 +182,6 @@ class PDAL_DLL PointView : public PointContainer
         getFieldInternal(dim, idx, buf);
     }
 
-    /*! @return a cumulated bounds of all points in the PointView.
```

Comment:
This is API change.

Reply:
What is the case to remove PointView::calculateBounds()? Can we keep the method and simply forward down to eigenutils or wherever?

Reply:
The case is only, when I wanted to add demean and transform to the PointView, it was suggested that calculateBounds could probably move out. I don't care either way.